style.css

```diff
@@ -0,0 +1,136 @@
+*{
+    margin:0;
+    padding:0;
+}
+header img{
+    height: 70px;
+    margin-left: 10%;
+}
+section header{
+    height: 30vh;
+    background-color: #171A29;
+}
+section header img{
+    float: left;
+    margin: 3vh 0 3vh 12%;
+    height: 24vh;
+}
+section header p{
+    display: flow-root;
+    padding: 100px 0px 100px 80px;
```

Comment:
Instead of this, use flex on header to vertically centre align both the image and the heading